app/tests/conftest.py

```diff
@@ -17,7 +17,7 @@ def new_user(test_client):
 @pytest.fixture(scope='function')
 def test_client():
     flask_app = create_app(config_name='testing')
-
+    flask_app.config['SECRET_KEY'] = '192b9bdd22ab9ed4d12e236c78afcb9a393ec15f71bbf5dc987d54727823bcbf'
```

Comment:
@bashaashma we could abstract this using env variables, just as used in the config.py file. This exposes our app secret key

Reply:
Oh okay sounds good. I can try that, but no worries that's not our app secret key. It's one I found online for testing.